.env-sample

```diff
@@ -0,0 +1 @@
+baseURL = '';
```

Comment:
Remove the space before and after the `=` sign. Also, remove the semi colon and single quote. `dotenv` will read it as part of the value.

It should be `baseURL=` instead

Reply:
Thanks for that observation @adesege I had missed that,
Feedback implemented